packages/react-ui-core/src/Menu/Menu.js

```diff
@@ -46,6 +47,7 @@ export default class Menu extends PureComponent {
   constructor(props) {
     super(props)
 
+    this.state = {}
```

Comment:
Why are you setting state as an empty object here and assigning it a value right below?